bin/pt-replica-find

```diff
@@ -0,0 +1,4819 @@
+binmode on STDOUT without the utf8 layer, and runs SET NAMES after
+connecting to MySQL.
+
+=item --config
+
+type: Array
+
+Read this comma-separated list of config files; if specified, this must be the
+first option on the command line.
+
+=item --database
+
+type: string; short form: -D
+
+Database to use.
+
+=item --defaults-file
+
+short form: -F; type: string
+
+Only read mysql options from the given file.  You must give an absolute
+pathname.
+
+=item --help
+
+Show help and exit.
+
+=item --host
+
+short form: -h; type: string
+
+Connect to host.
+
+=item --password
+
+short form: -p; type: string
+
+Password to use when connecting.
+If password contains commas they must be escaped with a backslash: "exam\,ple"
+
+=item --pid
+
+type: string
+
+Create the given PID file.  The tool won't start if the PID file already
+exists and the PID it contains is different than the current PID.  However,
+if the PID file exists and the PID it contains is no longer running, the
+tool will overwrite the PID file with the current PID.  The PID file is
+removed automatically when the tool exits.
+
+=item --port
+
+short form: -P; type: int
+
+Port number to use for connection.
+
+=item --recurse
+
+type: int
+
+Number of levels to recurse in the hierarchy.  Default is infinite.
+
+See L<"--recursion-method">.
+
+=item --recursion-method
+
+type: array; default: processlist,hosts
+
+Preferred recursion method used to find replicas.
+
+Possible methods are:
+
+  METHOD       USES
+  ===========  ==================
+  processlist  SHOW PROCESSLIST
+  hosts        SHOW REPLICAS (SHOW SLAVE HOSTS before MySQL 8.1)
+  none         Do not find replicas
+
+The processlist method is preferred because SHOW REPLICAS is not reliable.
+However, the hosts method is required if the server uses a non-standard
+port (not 3306).  Usually pt-replica-find does the right thing and finds
+the replicas, but you may give a preferred method and it will be used first.
+If it doesn't find any replicas, the other methods will be tried.
+
+=item --report-format
+
+type: string; default: summary
+
+Set what information about the replicas is printed.  The report format can be
+one of the following:
+
+=over
+
+=item * hostname
+
+Print just the hostname name of the replicas.  It looks like:
+
+  127.0.0.1:12345
+  +- 127.0.0.1:12346
+     +- 127.0.0.1:12347
+
+=item * summary
+
+Print a summary of each replica's settings.  This report shows more information
+about each replica, like:
+
+  127.0.0.1:12345
+  Version         5.1.34-log
+  Server ID       12345
+  Uptime          04:56 (started 2010-06-17T11:21:22)
+  Replication     Is not a slave, has 1 slaves connected
+  Filters
+  Binary logging  STATEMENT
+  Slave status
+  Slave mode      STRICT
+  Auto-increment  increment 1, offset 1
+  +- 127.0.0.1:12346
+     Version         5.1.34-log
+     Server ID       12346
+     Uptime          04:54 (started 2010-06-17T11:21:24)
+     Replication     Is a slave, has 1 slaves connected
```

Comment:
Unaligned with the rest, but coherent with the current states of things
Again, I can flag it for later, or skip

Reply:
Fixed